### DISCUSSION TITLE
Provide the feature to flip the screen while watching episodes

### DISCUSSION THREAD

**Discussion by Sjnn** [aniyomiorg/aniyomi#669] [Ideas]
Providing the feature to flip the screen while watching episodes because watching from one side is sometimes very difficult

**Comment by 12ohit** [comment]
If you mean being able to rotate the player then that feature is already added in beta version of Aniyomi and in future will make it to the stable version too, if you want to try it out right now then you can download it from here: https://github.com/jmir1/aniyomi-mpv-beta/releases/latest
